src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -37,6 +40,38 @@ public TypeNode type() {
   public abstract static class Builder {
     public abstract Builder setValue(String value);
 
-    public abstract StringObjectValue build();
+    public abstract StringObjectValue autoBuild();
+
+    public StringObjectValue build() {
+      // `\"` is added to the escaped string value for interpreting it correctly in file.
+      String value =
+          String.format("\"%s\"", StringValueEscaper.getInstance().escape(autoBuild().value()));
```

Comment:
Since `escape` is already a `public static` method, do we still need the `getInstance` call?

Reply:
Unfortunately `escape()` is public but not `static`, otherwise you are right, we can directly call it from Builder class.